admin/sql/create_tables.sql

```diff
@@ -89,7 +89,7 @@ CREATE TABLE dataset (
 CREATE TABLE dataset_class (
   id          SERIAL,
   name        VARCHAR NOT NULL,
-  description TEXT,
+  description TEXT    NOT NULL, -- NULL values are not accepted
```

Comment:
we should do the same with the `dataset` table above too. The comment here isn't necessary, we can already see from the NOT NULL keyword that nulls aren't accepted.
The update script should also update the dataset table

Reply:
The changes are done. The update script is also updated for making the similar changes to the dataset table